src/utilities/inputParsers.ts

```diff
@@ -30,7 +30,11 @@ export const parseInputMergeMethod = (): AllowedMergeMethods => {
 export const parseInputMergePreset = (): AllowedMergePresets | undefined => {
   const input = getInput('PRESET');
 
-  if (input.length === 0 || AllowedMergePresets[input] === undefined) {
+  if (input.length === 0) {
+    return undefined;
+  }
+
+  if (AllowedMergePresets[input] === undefined) {
     logWarning('PRESET value input is ignored because it is malformed.');
```

Comment:
@merlinnot I'll keep the warning here for when PRESET is present but not wel-fromed